cli/tests/program.rs

```diff
@@ -488,13 +488,27 @@ fn test_cli_program_deploy_with_authority() {
         program_data[..]
     );
 
-    // Set a new authority
+    // Set a new authority sign offline first
     let new_upgrade_authority = Keypair::new();
     config.signers = vec![&keypair, &upgrade_authority];
     config.command = CliCommand::Program(ProgramCliCommand::SetUpgradeAuthority {
```

Comment:
Since the main use-case for this is `SetUpgradeAuthorityChecked` with an offline signer, how about we change the test to exercise that behavior?
```suggestion
    // Set a new authority sign offline first
    let new_upgrade_authority = Keypair::new();
    config.signers = vec![&keypair, &upgrade_authority, &new_upgrade_authority];
    config.command = CliCommand::Program(ProgramCliCommand::SetUpgradeAuthorityChecked {
```

Reply:
If the main use-case is for `SetUpgradeAuthorityChecked`, should we keep the offline signing for `SetUpgradeAuthority`? Is there use case for this one?